packages/cni/cni.spec

```diff
@@ -29,7 +29,7 @@ Requires: %{_cross_os}iptables
 %build
 %set_cross_go_flags
 
-go build -buildmode=pie -ldflags="${GOLDFLAGS}" -o "bin/cnitool" %{goimport}/cnitool
+go build -ldflags="${GOLDFLAGS}" -o "bin/cnitool" %{goimport}/cnitool
```

Comment:
No fips version for this package?

Reply:
> No fips version for this package?

It doesn't import `crypto/tls`, so it doesn't need it. `oci-add-hooks` and `ecs-gpu-init` were the same way.